src/components/Canvas.tsx

```diff
@@ -73,8 +65,8 @@ export const Canvas: FC<Props> = ({
           height={height / zoom}
           opacity="0"
         />
-        {ShapesElement}
-        {DotsElement}
+        <Shapes shapes={shapes} />
+        <Dots dots={dots} />
```

Comment:
I swapped to declaring the components inline so I could add props to them. Was there a reason they were declared as props on the Canvas component originally

Reply:
Yes, the key feature of katachidraw is reusable RN component library and this allows full customization. And, it's where atoms shine.